.github/scripts/transformers_result_check.sh

```diff
@@ -0,0 +1,60 @@
+transformers_test=${1:-backbone}
```

Comment:
Let's drop this script and associated step for now. Please, save script somewhere since we might need it in the future.